relay-event-normalization/src/normalize/span/description/sql/mod.rs

```diff
@@ -698,6 +698,46 @@ mod tests {
         "UPDATE tbl SET foo = CASE WHEN .. THEN .. END"
     );
 
+    scrub_sql_test!(
+        duplicate_conditions,
+        r#"SELECT *
+        FROM a
+        WHERE a.status = %s
+        AND (
+            (a.id = %s AND a.org = %s)
+            OR (a.id = %s AND a.org = %s)
+            OR (a.id = %s AND a.org = %s)
+            OR (a.id = %s AND a.org = %s)
+        )"#,
+        "SELECT * FROM a WHERE status = %s AND ((id = %s AND org = %s))"
+    );
+
+    scrub_sql_test!(
+        duplicate_conditions_or,
+        r#"SELECT *
+        FROM a
+        WHERE a.status = %s
+        OR (
+            (a.id = %s OR a.org = %s)
+            OR (a.id = %s OR a.org = %s)
+            OR (a.id = %s OR a.org = %s)
+            OR (a.id = %s OR a.org = %s)
+        )"#,
+        "SELECT * FROM a WHERE status = %s OR ((id = %s OR org = %s))"
```

Comment:
@gggritso this is not exactly the placeholder you suggested, but it does keep the implementation simple. Let me know if it's good enough.

Reply:
Works for me 👍🏻 going to defer to @Zylphrex on this, since he found the original issue

Reply:
This looks fine to me as it covers the case I originally raised.